React-Native/en/1.4.12.md

```diff
@@ -0,0 +1,4 @@
+# [Success Criterion 1.4.12 - Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)
+
+Make sure a user can change and adapt the standard text spacing of the app's text elements by refraining from fixed heights.
+By using padding and margins properly a pixel perfect design is achievable without limiting the flexibility of the assistive technologies see [Success criterion - 1.4.5.md](1.4.5.md)
```

Comment:
1.4.5 is Images of Text, not relevant for 1.4.12? Or is is meant that when not using images of text, your layout can adapt better?